eli5/sklearn/explain_weights.py

```diff
@@ -422,3 +424,17 @@ def _features(target_id):
             method='linear model',
             is_regression=True,
         )
+
+
+@explain_weights.register(Pipeline)
```

Comment:
After https://github.com/TeamHG-Memex/eli5/pull/170 it should be just `@register`, so that it is registered both for global and sklearn-specific explain functions.